x-pack/plugins/observability_ai_assistant/public/mock.tsx

```diff
@@ -64,6 +65,8 @@ export const mockService: ObservabilityAIAssistantService = {
       navigate: () => {},
     } as unknown as SharePluginStart),
   register: () => {},
+  setApplicationContext: () => noop,
+  getApplicationContexts: () => [],
```

Comment:
What about calling this API `chatContext`? We might also want to refer to it internally as "consumer context" to distinguish it from "knowledge base context".

```suggestion
  setChatContext: () => noop,
  getChatContexts: () => [],
```

Reply:
See comment above, I don't think `chatContext` is the way to go but open to other alternatives.